.bazelrc

```diff
@@ -51,7 +51,6 @@ build:cuda --repo_env TF_NEED_CUDA=1
 # "compute" means we emit both cubin and PTX, which is larger but also forward compatible to future GPU generations.
 build:cuda --crosstool_top=@local_config_cuda//crosstool:toolchain
 build:cuda --@local_config_cuda//:enable_cuda
-build:cuda --@xla//xla/python:enable_gpu=true
```

Comment:
why is this one removed?

Reply:
gave error on enable_gpu not found when tried to build locally